### DISCUSSION TITLE
Support for Interest

### DISCUSSION THREAD

**Discussion by cbirkenbeul** [ghostfolio/ghostfolio#2224] [Ideas]
Hi,
maybe its possible to add interest as activity in the near future. I'm investing in P2P credits and there you get interest. I cannot use dividend for that, because it's not a value per share.
Thats also make it possible to add other types of accounts like "day money" accounts. I don't know the right word for the German "Tagesgeld". 
Thanks.

**Comment by JPFrancoia** [comment]
I'm willing to sponsor $50 for this feature and this one: https://github.com/ghostfolio/ghostfolio/issues/1950. These two features can probably be implemented at the same time, since it's just 2 new types of activities. For whoever implements the features.

**Reply by JPFrancoia** [reply]
The feature in #1950 has been implemented separately.
Still happy to sponsor the interest feature (this one) for $40.

**Comment by dtslvr** [comment]
Issue: https://github.com/ghostfolio/ghostfolio/issues/2353

**Comment by Loigzorn** [comment]
Hello ghostfolio community,
and hello @dtslvr,

Thank you for implementing the feature to add interest activities on the account basis.

I just enabled this feature in the settings by checking the experimental features in the settings.
Indeed I was able to create a new activity with interest as type of the activity, I just wonder how this could be presented to the user as I was not able to see the created activities in my overview or the analysis/allocation of my portfolio.

This, I think goes hand in hand with the topic of [discussion 2308](https://github.com/ghostfolio/ghostfolio/discussions/2308), as whether or not to display a sum of cash balance in one of the views.
